l1-contracts/src/core/Rollup.sol

```diff
@@ -105,4 +126,12 @@ contract Rollup is IRollup {
       revert Errors.Rollup__TimestampTooOld();
     }
   }
+
+  function _computePublicInputHash(
+    bytes calldata _header,
+    bytes32 _diffRoot,
+    bytes32 _l1ToL2MsgsHash
```

Comment:
When you pass in the params on line 76 you call these hashes txsHash, inHash. Would keep the terminology consistent with yellow paper so please update the occurrences of diffRoot and such.

Reply:
Oh, you are right, some moving around leftovers. Fixing now.